index.js

```diff
@@ -1,42 +1,60 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1;
+counter = 2; //re-assign values
+console.log(counter) 
 // // => 2
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
+console.log("%cQuestion 2", "color: red") 
+const name = "Chris" //when using const, cant change/overwrite name. use const when you know for a fact 
```

Comment:
Great! The key here was using `const` because it doesn't allow us to reassign our variable